lib/src/breakpoints_utils.dart

```diff
@@ -9,18 +9,18 @@ extension BreakpointExtensions on BuildContext {
   double get margin => materialSpacing;
 
   /// The size according to Material Design's with a specified [GutterType].
-  double gutter({GutterType type = GutterType.medium, double? scaleFactor}) {
+  double gutter({GutterType type = GutterType.medium, double scaleFactor = 0}) {
```

Comment:
Hmm this doesn't look right to me. This is what I was thinking this code would look like:
```dart
double gutter({GutterType type = GutterType.medium, double? scaleFactor}) {
  return ((scaleFactor ?? 1) * switch (type) {
    GutterType.tiny => materialSpacing / 4,
    GutterType.small => materialSpacing / 2,
    GutterType.medium => materialSpacing,
    GutterType.large => materialSpacing * 2,
    GutterType.extraLarge => materialSpacing * 4,
    GutterType.expand => double.infinity,
  });
}
  ```
  
  Does the above look right to you?

Reply:
Right. I can have a look tomorrow to mix the two ways? I do like how the scale is now set in the ctor, but we need something better for this...

Reply:
Yeah my sense is that either:
A. Scale and type should both be considered and multiply against each other (my strong preference as in niche places I do find myself wanting stuff like `1.5*GutterMedium`)
or
B. The method only takes in scaleFactor and scale factor is set according to the gutter type (not as ideal because we now lose the escape hatch)

Reply:
@caseycrogers How about with this change? This makes that possible out of the box.